devtools/cli/src/test/java/io/quarkus/cli/CliProjectJBangTest.java

```diff
@@ -51,9 +51,9 @@ public void testCreateAppDefaults() throws Exception {
                 CreateProjectHelper.DEFAULT_ARTIFACT_ID,
                 CreateProjectHelper.DEFAULT_VERSION);
 
-        Path javaMain = valdiateJBangSourcePackage(project, ""); // no package name
+        Path javaMain = validateJBangSourcePackage(project, ""); // no package name
```

Comment:
Same here, this change has nothing to do with this PR.

I won't mark all of them but all the unrelated changes needs to be done in another PR.

Reply:
Updated to remove unrelated changes.